.github/workflows/build-and-test.yaml

```diff
@@ -126,9 +126,9 @@ jobs:
             ./*.charm
           if-no-files-found: error
 
-  integration-test-only:
+  integration-test:
     name: Integration tests
-    if: inputs.scope == 'integration-test-only' || inputs.scope == 'all'
+    if: ${{ always() && (inputs.scope == 'integration-test-only' || inputs.scope == 'all') }}
```

Comment:
Mmh, I'm not sure about this.

With this, we would run integration tests even if charms fail to build, right?